internal/compiler/tests/syntax/lookup/conversion.slint

```diff
@@ -8,8 +8,11 @@ global Glob := {
 //                              ^error{Cannot convert between logical and physical length in a global component, because the scale factor is not known}
     property <float> ratio: allowed / 1phx;
 //                         ^error{Cannot convert between logical and physical length in a global component, because the scale factor is not known}
-    property <float> should_work: 45px / 8px;
+    property <length> converted_rem: 2rem;
+//                                  ^error{Cannot convert between rem and logical length in a global component, because the default font size is not known}    
```

Comment:
I think we might be able to get rid of this restriction now.

That said, when we will support several window, this may become a problem again.